src/database/migrations/V1__addUsers.ts

```diff
@@ -1,14 +1,15 @@
 import Knex from 'knex';
 
 export async function up(db: Knex) {
-  await db.schema.createTable('example_users', (table) => {
-    table.bigIncrements('id');
-    table.timestamps(true, true);
+  await db.schema.createTable('user', (table) => {
+    table.bigIncrements('id').primary();
     table.string('email').unique().notNullable();
-    table.string('hashed_password').notNullable();
+    table.timestamp('creation_time').nullable().index();
+    table.jsonb('personal_information').nullable();
+    table.jsonb('latest_address').nullable();
```

Comment:
haven't gotten there yet.. I'd start with this as it can be also indexable for text search (JSONB and more flexible). If we see an issue we can always create a new table for those. But with nested JSONB we have less tables to worry about + faster queries as the resources are inlined.